services/default-templates/blank.js

```diff
@@ -0,0 +1,20 @@
+export default {
+  name: '',
+  type: 'ExampleCredential',
+  fields: [{
+    type: 'h4',
+    gutter: true,
+    default: 'My Credential',
+    label: 'Title',
+  }, {
+    type: 'h5',
+    gutter: true,
+    default: '{name}',
+    label: 'Display Name',
+    jsonField: 'name',
+  }, {
+    type: 'body2',
+    default: 'Issued at: {date}',
+    label: 'Date',
```

Comment:
should issuer be a standard field too?

Reply:
its added when actually creating a certificate, this is just fields to display a pretty credential